.github/workflows/main.yml

```diff
@@ -41,13 +41,13 @@ jobs:
     
     - name: Setup Python MacOS
       run: |
-            wget https://www.python.org/ftp/python/3.9.2/python-3.9.2-macos11.pkg
-            sudo installer -verbose -pkg ./python-3.9.2-macos11.pkg -target /
-            echo "/Library/Frameworks/Python.framework/Versions/3.9/bin" >> $GITHUB_PATH
+            wget https://www.python.org/ftp/python/3.10.11/python-3.10.11-macos11.pkg
```

Comment:
why is this being changed?

Reply:
Not sure why this is showing different, this is what we have on master atm. The reason we have this change is is so that the package that had issues with different versions of mac is solved in Python versions 3.10+.